lib/downloader.rb

```diff
@@ -0,0 +1,35 @@
+# Defines how chromebrew downloads files.
+require_relative 'color'
+
+def notimplemented(package)
+  puts "#{package} has not been implemented as a downloader for Chromebrew yet. Defaulting to curl.".orange
+  @_crew_downloader = "curl -#LC -"
+  def download_file(url, filename)
+    system "#{@_crew_downloader} \'#{url}\' --output #{filename}"
+  end
+end
+
+case ENV["CREW_DOWNLOADER"]
```

Comment:
We can use `.to_s` so that the value is always a string even if it is not set

Reply:
That's why I have the `nil` in there.